httpserver/rust/src/httpserver.rs

```diff
@@ -132,6 +135,10 @@ pub struct HttpRequest {
     #[serde(with = "serde_bytes")]
     #[serde(default)]
     pub body: Vec<u8>,
+    /// Websocket Message as byte array. May be empty.
+    #[serde(with = "serde_bytes")]
+    #[serde(default)]
+    pub message: Vec<u8>
```

Comment:
My first thought here is that we could re-use the `body` payload so that we don't need to add another field